Math-Trainer/Math-Trainer/UIButton+Shadow.swift

```diff
@@ -9,6 +9,7 @@ import UIKit
 
 extension UIButton {
     func addShadow() {
+        self.layer.backgroundColor = UIColor.systemYellow.cgColor
```

Comment:
this is not a part of shadow 
remove pls

Reply:
Done, thanks.